### DISCUSSION TITLE
Issue with Socket

### DISCUSSION THREAD

**Discussion by noahstoner** [Finalgoal231/finalGoal#46] [Discuss about technical stack] [answered]
### Discussed in https://github.com/orgs/Finalgoal231/discussions/12

<div type='discussions-op-text'>

<sup>Originally posted by **shinevue** July  4, 2024</sup>
Hello, truestar.
Fix socket error that you made.
This is critical error.</div>

**Comment by creative2113** [comment, marked as answer]
Hello, @shinevue 
I am a full stack developer.
I can fix your error.
if you give me that, I will fix in a short time.
